Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue3548.cs

```diff
@@ -0,0 +1,82 @@
+﻿using Xamarin.Forms.Internals;
+using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Controls.Effects;
+using System.Threading.Tasks;
+
+#if UITEST
+using Xamarin.UITest;
+using NUnit.Framework;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 3548, "Cannot attach effect to Frame", PlatformAffected.iOS, NavigationBehavior.PushAsync)]
```

Comment:
```suggestion
	[Issue(IssueTracker.Github, 3548, "Cannot attach effect to Frame", PlatformAffected.Android, NavigationBehavior.PushAsync)]
```

Reply:
yeah.. my bad, i added for testing and forgot to rollback to android
txh